pkg/volume/csi/csi_client.go

```diff
@@ -129,7 +129,7 @@ func (c *csiDriverClient) AssertSupportedVersion(ctx grpctx.Context, ver *csipb.
 	c.versionSupported = supported
 
 	if !supported {
-		return fmt.Errorf("version %s not supported", verToStr(ver))
+		return fmt.Errorf("Expected version  %s. Version reported by CSI driver of %s is not supported", verToStr(ver), versToStr(vers))
```

Comment:
I think we want to use lower case to start returned Errorf strings as they are often wrapped in bigger error messages later.

Why are there two spaces between "Expected version" and "%s", is it a typo?

The error message seems a little backwards to me. Isn't "vers" a list of the versions supported by the driver (which you have stated here is "version reported by CSI driver NOT supported"). 

Also "Expected version" seems a little bit vague, maybe "spec specified version" or something of that nature? This is just personal preference though.